app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external services. This catalog is the single source of truth for your organization’s service inventory and their dependencies.
+
+{% mermaid %}
+graph TD
+    subgraph "Built-In Integrations"
+        D[{{site.base_gateway}}]
+        E[{{site.mesh_product_name}}]
+        F[{{site.konnect_short_name}} Analytics]
+        G[API Products]
+    end
+
+    subgraph "External Integrations"
+        H[GitHub]
+        I[PagerDuty]
+    end
+
+    D -->A[{{site.konnect_short_name}} Service Catalog]
+    E -->A
+    F -->A
+    G -->A
+
+    H -->A
+    I -->A
+{% endmermaid %}
+
+> Figure 1: This diagram shows how you can pull services from built-in integrations (like {{site.base_gateway}}, {{site.mesh_product_name}}, {{site.konnect_short_name}} Analytics, and API Products) and external services (like from GitHub and PagerDuty) into the Service Catalog. This consolidates all your organization's services and service health metrics into a single view.
```

Comment:
It's not only health metrics that Service Catalog aggregates. Rather, Service Catalog consolidates a data from a multitude of sources that developers need to ensure their service stays up and running. These include: code repositories (GitHub/GitLab/Bitbucket), incident management (PagerDuty/OpsGenie/Incident.io), observability (Datadog/New Relic), security (Snyk/Sonaqube), CI/CD (CircleCI/Jenkins), infrastructure (Kubernetes/AWS/Azure/GCP), and so on.